opm/autodiff/BlackoilSolventModel.hpp

```diff
@@ -123,7 +123,7 @@ namespace Opm {
         using Base::has_vapoil_;
         using Base::param_;
         using Base::use_threshold_pressure_;
-        using Base::threshold_pressures_by_interior_face_;
+        using Base::threshold_pressures_by_connection_;
```

Comment:
The same changes must be made to the polymer model, fortunately that can now be done in the same PR, after we combined the repos.